### ISSUE TITLE
Published plugin misses its oozie-jaxb dependency

### ISSUE THREAD

**Issue by radimk** [linkedin/linkedin-gradle-plugin-for-apache-hadoop#53]
Applying the plugin using the `buildscript` block as shown on https://plugins.gradle.org/plugin/com.linkedin.gradle.hadoop.HadoopPlugin fails with message like:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'mapred-samples'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not find linkedin-gradle-plugin-for-apache-hadoop:oozie-jaxb:0.7.9.
     Searched in the following locations:
         https://plugins.gradle.org/m2/linkedin-gradle-plugin-for-apache-hadoop/oozie-jaxb/0.7.9/oozie-jaxb-0.7.9.pom
         https://plugins.gradle.org/m2/linkedin-gradle-plugin-for-apache-hadoop/oozie-jaxb/0.7.9/oozie-jaxb-0.7.9.jar
     Required by:
         :mapred-samples:unspecified > gradle.plugin.linkedin-gradle-plugin-for-apache-hadoop:hadoop-plugin:0.7.9
```

**Comment by convexquad** [comment]
@radimk Thanks for reporting this. At LinkedIn, our internal cut of Gradle automatically adds a repository that has all the jars we need, and it's somewhat hard to work around it, so I missed it. I'll try to fix the issue and publish a new version to plugins.gradle.org before the end of the week!

**Comment by radimk** [comment]
Thanks, this would be awesome.

**Comment by convexquad** [comment]
I have published the dependent jars for the Hadoop Plugin to Bintray (jcenter), but the Maven coordinates aren't quite right. I'm still going to have to fiddle with it for a bit more...

**Comment by convexquad** [comment]
I figured out how to get the right Maven coordinates to publish to Bintray. It turns out that you then in turn link your Bintray repo to JCenter. I just submitted a request to have them linked - once it is fulfilled, I'll try re-publishing to plugins.gradle.org and we can see if it all works.

**Comment by convexquad** [comment]
https://bintray.com/convexquad/maven/linkedin-gradle-plugin-for-apache-hadoop/view

**Comment by radimk** [comment]
Thanks, it works now with 0.8.6 using snippet from plugins.gradle.org.

```buildscript {
  repositories {
    maven {
      url "https://plugins.gradle.org/m2/"
    }
  }
  dependencies {
    classpath "gradle.plugin.com.linkedin.hadoop-plugin:hadoop-plugin:0.8.6"
  }
}
```

**Comment by convexquad** [comment]
That was impressive... you commented literally one minute after I published version 0.8.6 with the corrections that make it work!

Just FYI, the documentation available at https://github.com/linkedin/linkedin-gradle-plugin-for-apache-hadoop/wiki needs to be refreshed just a little bit for a few new minor features we added. However, it should still be accurate for all the old features.

**Comment by radimk** [comment]
LOL it was just by chance.
